crates/solver-types/src/api.rs

```diff
@@ -840,8 +863,7 @@ impl PostOrderRequest {
 				crate::utils::eip712::reconstruct_3009_digest_from_quote(quote)
 			},
 			OifOrder::OifResourceLockV0 { .. } => {
```

Comment:
nit; can be replaced with `_ => Err...` and that encapsulates all types except for the above